Automattic/wp-calypso
# Split plans selectors

This PR split plans selectors into two modules, based on imports. This isolates the dependency on `lib/plans`, limiting it to the single selector that makes use of it.

This PR is working towards the end goal of removing the dependency on `lib/plans` from the critical path, which may yet turn out to be impossible.

#### Changes proposed in this Pull Request

* Split plans selectors into two modules, based on imports

#### Testing instructions

This PR doesn't change any functional code, it simply moves some exports around. As such, a correct build and passing automated tests should be enough to ensure that nothing got broken.
